### PR TITLE
feat: blockchain node

### PR DESCRIPTION
**Pull request by skiv71**
This PR adds a new docker image which allows a user to spin up a local RPC server, useful to bootstrap the graph node during sync, but can also be  run indefinitely.

### CODE REVIEW

File: request-subgraph-storage/.resources/apt.txt
[the remark this answers is not in the document]
Comment (MantisClone) [reply request-subgraph-storage/.resources/apt.txt:1]: I switched the target branch to `feat/easypanel` :white_check_mark:

File: request-node/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG NODE_VERSION=16.15.0-alpine3.15
+ARG NODE_VERSION=18.19.0-alpine3.19
```
Comment (MantisClone) [inline request-node/Dockerfile:1]: Please revert this change and create a separate PR for it, if needed.

File: request-chain/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Request Chain
+
+## Description
+
+This image provides the means to deploy an ethereum node with both execution client [Nethermind](https://www.nethermind.io/nethermind-client) & consensus client [Teku](https://consensys.io/teku).
```
[the remark this answers is not in the document]
Comment (skiv71) [reply request-chain/README.md:5]: As I've mentioned before, this docker image provides a service where the dependencies share resources and need to be run in a concurrent / controlled manner. With regard to a single PID and reporting back to the instigating docker runtime, that issue can be mitigated with a PID watcher / patch

[the remark this answers is not in the document]
Comment (MantisClone) [reply request-chain/README.md:5]: Based on the [Teku docs](https://docs.teku.consensys.io/24.2.0/get-started/install/run-docker-image#run-teku-using-docker-compose), it's possible to run the consensus and execution clients in separate docker images via a docker-compose script.

**Comment by MantisClone** [comment]
It was decided to archive the blockchain node docker image because it was only used to bootstrap the Graph node PostgreSQL database and is no longer used in production.
